homeassistant/components/tibber/__init__.py

```diff
@@ -60,7 +82,34 @@ async def retry_setup(now):
         _LOGGER.error("Failed to login. %s", exp)
         return False
 
-    for component in ["sensor", "notify"]:
-        discovery.load_platform(hass, component, DOMAIN, {CONF_NAME: DOMAIN}, config)
-
+    for component in PLATFORMS:
+        hass.async_create_task(
+            hass.config_entries.async_forward_entry_setup(entry, component)
+        )
+
+    # set up notify platform, no entry support for notify component yet,
+    # have to use discovery to load platform.
+    hass.async_create_task(
+        discovery.async_load_platform(
+            hass, "notify", DOMAIN, {CONF_NAME: DOMAIN}, hass.data.get(DATA_HASS_CONFIG)
```

Comment:
Key always exist, use square brackets
```suggestion
            hass, "notify", DOMAIN, {CONF_NAME: DOMAIN}, hass.data[DATA_HASS_CONFIG]
```